app/services/admin/depositors_search.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Admin
+  # Return a list of depositors based on druids
+  class DepositorsSearch
+    def self.call(...)
+      new(...).call
+    end
+
+    def initialize(form:)
+      @form = form
+    end
+
+    def call
+      form.druid_list.map do |druid|
+        work = Work.find_by!(druid:)
```

Comment:
There's probably a faster query that only retrieves the email address.

Also, might be able to do this as a single query instead of a query for each work.

Reply:
I changed this so that it issues two queries total. Maybe worth another look once I push up changes.